.github/actions/check_artifact_exists/dist/index.js

```diff
@@ -91,11 +79,26 @@ async function main() {
     });
     console.log("==> Repo:", owner + "/" + repo);
 
-    const goodArtifacts = await getGoodArtifacts(client, owner, repo, name);
-    console.log("==> goodArtifacts:", goodArtifacts);
+    let goodArtifacts = [];
+    if (localCacheIndexPath.length > 0) {
+        try {
+            const index = JSON.parse(await fs.readFile(localCacheIndexPath, "utf8"));
+            goodArtifacts = getGoodArtifactsFromLocalIndex(index, name);
+            console.log("==> goodArtifactsFromLocalIndex:", goodArtifacts);
+        } catch (error) {
+            // Missing/corrupt local index? Swallow error and fallback on API below
```

Comment:
I think we would want some warning at least in case things goes bad, having an idea that:
 - there was an error
 - what was the error code / message

could help diagnose